charon/pkgs/npm.py

```diff
@@ -102,6 +111,10 @@ def handle_npm_uploading(
     )
     logger.info("Files uploading done\n")
 
+    logger.info("Start uploading manifest to s3")
+    client.upload_manifest(manifest_name, manifest_full_path, target, manifest_bucket_name)
+    logger.info("Manifest uploading is done\n")
```

Comment:
I think this step can be skipped with a log if manifest_bucket_name is not specified.